src/Kovan-DssSpell.sol

```diff
@@ -29,44 +39,254 @@ contract SpellAction {
     //  against the current release list at:
     //     https://changelog.makerdao.com/releases/kovan/active/contracts.json
 
-    ChainlogAbstract constant CHANGELOG = ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
+    ChainlogAbstract constant CHANGELOG =
+        ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
+
+    /// UNI-A
+    address constant UNI                = 0x0C527850e5D6B2B406F1d65895d5b17c5A29Ce51;
+    address constant MCD_JOIN_UNI_A     = 0xb6E6EE050B4a74C8cc1DfdE62cAC8C6d9D8F4CAa;
+    address constant MCD_FLIP_UNI_A     = 0x6EE8a47eA5d7cF0C951eDc57141Eb9593A36e680;
+    address constant PIP_UNI            = 0xe573a75BF4827658F6D600FD26C205a3fe34ee28;
+    bytes32 constant ILK_UNI_A          = "UNI-A";
+
+    // RENBTC-A
+    address constant RENBTC             = 0xe3dD56821f8C422849AF4816fE9B3c53c6a2F0Bd;
+    address constant MCD_JOIN_RENBTC_A  = 0x12F1F6c7E5fDF1B671CebFBDE974341847d0Caa4;
+    address constant MCD_FLIP_RENBTC_A  = 0x2a2E2436370e98505325111A6b98F63d158Fedc4;
+    address constant PIP_RENBTC         = 0x2f38a1bD385A9B395D01f2Cbf767b4527663edDB;
+    bytes32 constant RENBTC_ILK         = "RENBTC-A";
+
+    // DC IAM
+    address constant MCD_IAM_AUTO_LINE = 0x0D0ccf65cED62D6CfC4DA7Ca85a0f833cB8889E4;
 
-    address constant MCD_ADM            = 0x27E0c9567729Ea6e3241DE74B3dE499b7ddd3fe6;
-    address constant VOTE_PROXY_FACTORY = 0x1400798AA746457E467A1eb9b3F3f72C25314429;
+    // decimals & precision
+    uint256 constant public THOUSAND    = 10 ** 3;
+    uint256 constant public MILLION     = 10 ** 6;
+    uint256 constant public WAD         = 10 ** 18;
+    uint256 constant public RAY         = 10 ** 27;
+    uint256 constant public RAD         = 10 ** 45;
+
+    // Many of the settings that change weekly rely on the rate accumulator
+    // described at https://docs.makerdao.com/smart-contract-modules/rates-module
+    // To check this yourself, use the following rate calculation (example 8%):
+    //
+    // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
+    //
+    uint256 constant public THREE_PERCENT_RATE = 1000000000937303470807876289;
+    uint256 constant SIX_PERCENT_RATE          = 1000000001847694957439350562;
```

Comment:
I think we should remove the public getter for three percent.

Reply:
done